src/components/common/ConfirmModal.tsx

```diff
@@ -38,6 +38,11 @@ const dialogStyles = makeStyles({
     height: 48,
     width: 107,
   },
+  dialogConfirm: {
+    color: Colours.Secondary,
+    height: 48,
+    width: 107,
```

Comment:
Is this the same as `px`?

Reply:
not too sure, I just copied from above lol

Reply:
@JasonYG please convert all values without units to `px` to keep things consistent

Reply:
i just verified that all the numbers without units default to `px`.

Reply:
let's add the units in for the sake of consistency please - for all values that don't have units especially in this file (not just these 2)
```suggestion
    height: '48px',
    width: '107px',
```